### PR TITLE
Lf 2374 allow modification of due date on read only task view

### PR DESCRIPTION
**Pull request by fsayyid**
Added a pencil edit icon for the due date in the read only task view, which triggers the update due date modal to appear. Only those with admin roles can see the pencil icon. The due date can only be edited when the task is not completed or abandoned. 

Story: [LF-2374](https://lite-farm.atlassian.net/browse/LF-2374)

### CODE REVIEW

File: packages/webapp/src/components/Task/TaskReadOnly/index.jsx [hunk 1]
```diff
@@ -160,23 +163,14 @@ export default function PureTaskReadOnly({
...
-        value={date}
-        label={dateLabel}
-        disabled
-      />
+      <div className={styles.assigneeContainer} style={{ marginBottom: '40px' }}>
+        <Input type={'date'} value={date} label={dateLabel} disabled />
+        {isCurrent && isAdmin && (
+          <BiPencil className={styles.pencil} onClick={(_) => setShowDueDateModal(true)} />
+        )}
+      </div>
```
Comment (seinnlee) [inline packages/webapp/src/components/Task/TaskReadOnly/index.jsx:171]: Can we change the spacing and indentation to be more like line 155~164?

**Review by seinnlee — APPROVED** [review]
Functionally looks good to me! Just minor comment

File: packages/webapp/src/components/Task/TaskReadOnly/index.jsx (hunk 1 above)
Comment (fsayyid) [reply packages/webapp/src/components/Task/TaskReadOnly/index.jsx:171]: Screen Shot 2022-05-09 at 4 28 52 PM

**Review by smattingly — APPROVED** [review]
:+1:

**Pull request merged**